app/templates/direct-award/results.html

```diff
@@ -68,7 +67,6 @@
               "title": "Download search results as comma-separated values",
               "link": url_for('direct_award.download_results', framework_framework=framework.framework, project_id=project.id, filetype='csv'),
               "file_type": "CSV",
-              "download": "True",
```

Comment:
So if the 'documents' template has a feature that doesn't work, potentially we should be taking it out of there?

Reply:
the feature itself works, we may be using it for other external links which we cannot control the headers of.

Reply:
Hmmm, if it results in untestable code, then we should at least be sticking a comment in there warning people about that?